droidlet/interpreter/craftassist/mc_interpreter.py

```diff
@@ -285,7 +290,7 @@ def handle_fill(self, agent, speaker, d) -> Tuple[Any, Optional[str], Any]:
         location, _ = self.subinterpret["specify_locations"](self, speaker, mems, steps, reldir)
 
         # Get nearby holes
-        holes = heuristic_perception.get_all_nearby_holes(agent, location, self.block_data)
+        holes = self.get_all_holes_fn(agent, location, self.block_data)
```

Comment:
Lets mark this to be fixed/put in reasoning/run by agent perception.  I think this is one of the more egregious breakages of the rule that interpreter shouldn't perceive, but rather get perception from memory.  we can fix on the next pass when we make interpreter not a dialogue object; and it can ask the agent to do this perception (and more generally set up an "on demand perception" pathway where interpreter can ask agent to run a perception module and place results in memory)